src/Auth0Client.ts

```diff
@@ -695,6 +696,31 @@ export default class Auth0Client {
     return !!user;
   }
 
+  /**
+   * ```js
+   * await auth0.buildLogoutUrl(options);
+   * ```
+   *
+   * Clears the application session and builds an `/v2/logout` URL for logout using
+   * the parameters provided as arguments.
+   * @param options
+   */
+  public buildLogoutUrl(options: LogoutUrlOptions = {}): string {
+    if (options.client_id !== null) {
+      options.client_id = options.client_id || this.options.client_id;
+    } else {
+      delete options.client_id;
+    }
```

Comment:
I don't think this would yield the desired results. As `buildLogoutUrl` can be called directly without using `logout`, if this is removed from `buildLogoutUrl` it would not always result in the expected `client_id` being added onto the logout URL parameters.

The only way around it would be to perform these checks in each function. Am I misinterpreting something?

Reply:
Let's keep this here but remove it from the logout method.